plugins/groovy/groovy-psi/src/org/jetbrains/plugins/groovy/lang/documentation/GroovyPresentationUtil.java

```diff
@@ -94,12 +94,11 @@ else if (typePresentation == TypePresentation.CANONICAL) {
       if (!structural.isEmpty()) {
         builder.append(".");
         String[] array = ArrayUtilRt.toStringArray(structural);
-        if (array.length > 1) builder.append("[");
-        for (int i = 0; i < array.length; i++) {
-          if (i > 0) builder.append(", ");
-          builder.append(array[i]);
+        if (array.length > 1) {
```

Comment:
This change is incorrect: originally if array.length == 1, then the array[0] is still added (without brackets).

Reply:
Indeed, you are right!